MC/run/PWGEM/runHFToDielectrons_pp.sh

```diff
@@ -18,16 +18,16 @@ NWORKERS=${NWORKERS:-8}
 NTIMEFRAMES=${NTIMEFRAMES:-1}
 
 #generate random number
-RNDSIG=$((1 + $RANDOM % 100))
+RNDSIG=$((0 + $RANDOM % 100))
```

Comment:
you could even remove the `0 +` from the calculation. That does not do anything

Reply:
Should be done now.

Reply:
great! Let's wait for the CI and then I think we can merge it.